src/htm/algorithms/Connections.hpp

```diff
@@ -400,10 +400,10 @@ class Connections : public Serializable
    */
   void computeActivity(std::vector<SynapseIdx> &numActiveConnectedSynapsesForSegment,
                        std::vector<SynapseIdx> &numActivePotentialSynapsesForSegment,
-                       const std::vector<CellIdx> &activePresynapticCells);
+                       const std::vector<CellIdx> &activePresynapticCells) const;
 
   void computeActivity(std::vector<SynapseIdx> &numActiveConnectedSynapsesForSegment,
-                       const std::vector<CellIdx> &activePresynapticCells);
+                       const std::vector<CellIdx> &activePresynapticCells) const;
```

Comment:
Connections::computeActivity can be made const. With the "fake" (mutable) that for `timeseries_` the curent/previousUpdates are mutable & not locked (I may use mutex, or just let it be for now)